stock_picking_send_by_mail/models/stock_picking.py

```diff
@@ -6,30 +6,28 @@
 
 
 class StockPicking(models.Model):
-    _inherit = "stock.picking"
+    _inherit = 'stock.picking'
```

Comment:
Why changing to single quote? 
OCA repos use pre-commit with some prettyfier and so on that requires double quote. 
Don't change it or assure that the pre-commit works when working with OCA